src/components/DpDataTable/DpDataTable.vue

```diff
@@ -497,6 +497,23 @@ export default {
       return tableRow.$el.classList.add('is-hovered-content')
     },
 
+    /**
+     * This method is used to escape special characters in a string.
+     * It specifically targets spaces and plus signs.
+     *
+     * @param {string} string - The string to be processed.
+     * @returns {string} - The processed string with special characters escaped.
+     */
+    escapeSpecialCharacters (string) {
+      return string.replace(/[\s+]/g, (match) => {
+        if (match === ' ') {
+          return '\\s*';
```

Comment:
I actually don't now what "You" (the person before You) wants to archive with that. Can You give me an example input with the expected output?

Reply:
I thought it was used to escape whitespaces eg. `Test test`.

Reply:
That doesn't make sense to me. I could imagine that we want to replace special whitespaces to regular whitespaces. That would be something like `.replace(/\s/g, ' ')` . A bit the other way round ;-)

Reply:
if so  i suggest something like
```
 escapeSpecialCharacters (string) {
      return string
        .replace(/(\s)/g, ' ')
        .replace(/([+])/g, '\\$&') // here we can add other special chars if necessary without too much adjustment
```

Reply:
Done in [064afd7](https://github.com/demos-europe/demosplan-ui/pull/891/commits/064afd77bc01700fbf76a78d16846774eb4c1b73)